CMakeLists.txt

```diff
@@ -29,20 +29,24 @@ cmake_minimum_required(VERSION 3.11)
 
 set(PROJECT smartDeviceLinkCore)
 project(${PROJECT})
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH}
-    "${CMAKE_SOURCE_DIR}/tools/cmake/modules/")
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    "${CMAKE_SOURCE_DIR}/tools/cmake/modules/"
+)
 
 include("${CMAKE_CURRENT_LIST_DIR}/cmake/helpers/dependencies.cmake")
 include("${CMAKE_CURRENT_LIST_DIR}/tools/cmake/helpers/install_rules.cmake")
 include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/platform.cmake)
 include(
-  "${CMAKE_CURRENT_LIST_DIR}/tools/cmake/helpers/build_config_generator.cmake")
+    "${CMAKE_CURRENT_LIST_DIR}/tools/cmake/helpers/build_config_generator.cmake"
+    
```

Comment:
@vilyukh redundant empty line.

Reply:
@BSolonenko Since it breaks the command into tokens and the line break character can be in another token.
Therefore, in some cases there is a double line break. It is not possible to fix this issue since it is necessary to completely change the text processing logic of the command. That can lead to regression.

Reply:
@BSolonenko You can correct it, should run **cmake-format** with follow argument: _--dangle-parens 1_ or add in config-file as _dangle_parens: True_

Reply:
@dkandyba please make this default

Reply:
@dkandyba please make this default